src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -8,7 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <UseECMAKey>true</UseECMAKey>
     <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp'">$(DefineConstants);FEATURE_ZLIB</DefineConstants>
-    <DefineConstants>$(DefineConstants);netcoreapp20</DefineConstants>
+    <DefineConstants>$(DefineConstants);netstandard20</DefineConstants>
```

Comment:
I would remove this #ifdef given it isn't actually conditioned at all in this project.